### PR TITLE
Log forward model jobs

### LINKED ISSUE #1992: Log usage of forward models jobs
We should log which forward model jobs (name of jobs) are being run by job dispatch.

### PR DESCRIPTION
**Pull request by dafeda**
**Issue**
Resolves #1992

### CODE REVIEW

File: ert_gui/gert_main.py [hunk 1]
```diff
@@ -33,22 +37,27 @@
...
 
 def run_gui(args):
     app = QApplication([])  # Early so that QT is initialized before other imports
     app.setWindowIcon(resourceIcon("application/window_icon_cutout"))
     res_config = ResConfig(args.config)
+
+    # Create logger inside function to make sure all handlers have been added to the root-logger.
+    logger = logging.getLogger(__name__)
+    logger.info(
+        "Running ERT GUI...",
```
Comment (oyvindeide) [inline ert_gui/gert_main.py:55]: We are already logging this information else where, should we change the wording here?

**Review by oyvindeide — COMMENTED** [review]
Looks good, but can you reword the commit? We are logging the forward model jobs, not the workflow jobs.

File: ert_gui/gert_main.py (hunk 1 above)
Comment (dafeda) [reply ert_gui/gert_main.py:55]: "Logging forward model jobs when starting GUI"?

Comment (oyvindeide) [reply ert_gui/gert_main.py:55]: Not very important, but was thinking we did not need to GUI information, but will leave it up to you.

**Review by oyvindeide — APPROVED** [review]
LGTM!

**Comment by oyvindeide** [comment]
test this please

**Comment by dafeda** [comment]
Jenkins test this please

**Pull request merged**